src/app/models/test.type.ts

```diff
@@ -21,4 +26,13 @@ export interface TestType {
   name: string;
   certificateLink: string;
   testResult: string;
+  testExpiryDate: Date;
```

Comment:
Type should be string. this is what Backend sends 'YYYY-MM-DD'

Reply:
![image](https://user-images.githubusercontent.com/55185857/77894700-52decb80-727e-11ea-9b15-cc25946db296.png)

Reply:
@alleypa not always

Reply:
that fine Irina, I think better to still treat it as string. The conversion will just fine